src/yang/ietf-babel.yang

```diff
@@ -101,32 +112,31 @@ module ietf-babel {
 
   feature hmac-sha256-supported {
     description
-      "This implementation supports hmac-sha256 MAC algorithm.";
+      "This implementation supports the hmac-sha256 MAC algorithm.";
     reference
       "RFC 8967: MAC authentication for Babel Routing
        Protocol.";
   }
 
   feature blake2s-supported {
     description
-      "This implementation supports blake2s MAC algorithms.
-       Specifically, BLAKE2-128 is supported.";
+      "This implementation supports blake2s MAC algorithms.";
     reference
       "RFC 8967: MAC authentication for Babel Routing
        Protocol.";
   }
 
   feature x-509-supported {
     description
-      "This implementation supports x-509 certificate type.";
+      "This implementation supports the x-509 certificate type.";
     reference
       "RFC 8968: Babel Routing Protocol over Datagram
        Transport Layer Security.";
   }
 
   feature raw-public-key-supported {
     description
-      "This implementation supports raw-public-key certificate
+      "This implementation supports the RawPubliKey certificate
```

Comment:
s/RawPubliKey/Raw Public Key/

Reply:
I also notice that this is the only one you changed away from an all lower case term that resembled the YANG feature name toward the way the term is properly spelled. Should others be HMAC-SHA256, BLAKE2s, and X.509 for consistency?

Reply:
Ben suggested that the term was RawPublicKey (one word). But I see it being referenced as "Raw Public Key". Will switch it back.

For the others, I have changed them to HMAC-SHA256, BLAKE2s, and X.509 for it to be consistent.